en/presentation/index.html

```diff
@@ -132,19 +132,23 @@ <h3 style="color:#1166ad;">An Open Source Geospatial GNU/Linux Distribution</h3>
             </section>
             <section>
                <section>
+
                   <h2>Open Source Geospatial <br/> Foundation (OSGeo)</h2>
-                  <p align="left">Since 2006 A Non Profit Umbrella for:</p>
+                  <p align="left">Non Profit for geospatial community, fostering Open approach to:</p>
                   <ul>
-                     <li>GeoSpatial Free and Open Source Software</li>
+                     <li>Software</li>
```

Comment:
This removes Free Software from the presentation. I strongly disagree with only mentioning open approach.